src/WebJobs.Extensions.DocumentDB/DocumentDBAttribute.cs

```diff
@@ -64,7 +64,7 @@ public DocumentDBAttribute(string databaseName, string collectionName)
         /// Optional. A string value indicating the app setting to use as the DocumentDB connection string, if different
         /// than the one specified in the <see cref="DocumentDBConfiguration"/>.
         /// </summary>
-        public string ConnectionString { get; set; }
+        public string ConnectionStringSetting { get; set; }
```

Comment:
Those are actual ConnectionStrings. I sent a mail with the breakdown of the pattern we all use, but to simplify -- the Configuration classes take all real values, the attributes all take Settings.

---
In reply to: [65440424](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/108#discussion_r65440424) [](ancestors = 65440424)